pkg/c3/util.h

```diff
@@ -0,0 +1,82 @@
+#ifndef C3_UTIL_H
+#define C3_UTIL_H
+
+#include <stdio.h>
+
+#include "types.h"
+#include "defs.h"
+
+c3_w _c3_printcap_mem_w (FILE *fil_u, c3_w wor_w, const c3_c *cap_c);
+c3_z _c3_printcap_mem_z(FILE *fil_u, c3_z byt_z, const c3_c *cap_c);
+
+
```

Comment:
There are some Form Feed (U+000C) characters interspersed here and in other places. We should get rid of them.

Reply:
Oh ok, I just haven't seen them elsewhere in our codebase. Feel free to disregard.

Reply:
They certainly might not be anywhere else, but I don't think they hurt.